src/commons/index.js

```diff
@@ -25,9 +25,15 @@ module.exports = {
   get withScrollEnabler() {
     return require('./withScrollEnabler').default;
   },
+  get useScrollEnabler() {
+    return require('./useScrollEnabler').default;
+  },
   get withScrollReached() {
     return require('./withScrollReached').default;
   },
+  get useScrollReached() {
+    return require('./useScrollReached').default;
+  },
```

Comment:
we recently added a new Hooks folder, I suggest moving the hooks there..

Reply:
Done